.github/workflows/emberjs-addons-tag-release-publish.yml

```diff
@@ -0,0 +1,85 @@
+# GitHub Actions documentation:
+# https://docs.github.com/en/actions
+
+name: Create new `git tag`, create new GitHub release and publish to NPM
```

Comment:
Should we rename is `js-tag-release-publish`
If I'm correct we don't have any EmberJS related code in this workflow?

Reply:
Ah true true, we can do that indeed, that would require an update to the CODEOWNERS file.

Following your suggestion, what about this @GreatWizard :

- Move the workflow `.github/workflows/emberjs-addons-ci.yml`
  to `.github/workflows/js/emberjs-addons-ci.yml`

- Move the workflow `.github/workflows/emberjs-addons-tag-release-publish.yml`
  to `.github/workflows/js/tag-release-publish.yml`

- Update the CODEOWNERS to make the Tribe owner of the pattern:

```
/.github/workflows/js/                  @peopledoc/tribe-js
```

Reply:
We can't put workflows into subdirectories è_é.

See commit b4c7f08d0cfe062afadeeb1c9b8a947f256296d6 .